src/components/atoms/ImageComponent/Image.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+
+import './Image.scss';
+
+
+export const ImageComponent = ({ src, size, onClick }) => {
+    return (
+
+        <img src={`${src}`} clasName={`atom__image atom_image_${size}`} onClick={() => imageClick()}></img>
+
+    );
+};
+
+ImageComponent.propTypes = {
+    size: PropTypes.oneOf(['medium', 'large']),
+    onClick: PropTypes.func
+};
+
+ImageComponent.defaultProps = {
+    size: 'medium',
+    onClick: () => undefined
```

Comment:
we don't need the defaultProp here I think, in this case the image just won't be clickable